lib/validateMessage.js

```diff
@@ -63,6 +63,21 @@ exports.validateMessage = function validateMessage(raw) {
     var type = match[3];
     var scope = match[4];
     var subject = match[5];
+    config.scopes = config.scopes || {};
+    var validateScopes = config.scopes.validate || false;
+    var multipleScopesAllowed = config.scopes.multiple || false;
+    var allowedScopes = config.scopes.allowed || '*';
+    var scopeRequired = config.scopes.required || false;
+    var scopes = scope ? scope.split(',') : [];
+    var validateScope = (item) => {
```

Comment:
I don't think that we've ever supported `0.10.x`. But I think we can drop `0.12.x` too since it's no longer officially supported by NodeJS anyway.

As far as concise-ness, I'm not at all interested in making things concise. Let's focus on making things readable/maintainable for future contributors.